wormhole-connect/src/routes/operator.ts

```diff
@@ -238,7 +239,8 @@ export class Operator {
             sourceChain,
             destChain,
           );
-          if (isSupported) {
+          const isIlliquid = this.isIlliquidDestToken(token, destChain);
```

Comment:
IMO it's not necessary to add this here as a special case. instead I would put this check inside the `isSupported` method for the route(s) we're concerned with (token bridge & ETH bridge?). this kind of specific logic doesn't belong in the top level routes operator.

`isSupported` already checks that the tokens are supported - this could just be another level on that check.

in the [new Route API](https://github.com/wormhole-foundation/connect-sdk/pull/216/files#diff-49618d9f838bc1dfa71b211e57e3556cab6da7a720637c898293fdcf2e7bef19R22) that @barnjamin and I are working on, we expect all logic to be contained within `isSupported` (and `isAvailable` for routes using a relayer)

Reply:
changed